setup.py

```diff
@@ -179,31 +178,7 @@ def get_extensions():
     return [erfa_ext]
 
 
-try:
-    with open('erfa/_dev/scm_version.py') as fd:
-        source = fd.read()
-except FileNotFoundError:
-    guess_next_dev = None
-else:
-    import types
-    scm_version = types.ModuleType('scm_version')
-    scm_version.__file__ = 'erfa/_dev/scm_version.py'
-    code = compile(source, scm_version.__file__, 'exec')
-    try:
-        exec(code, scm_version.__dict__)
-    except ImportError:
-        guess_next_dev = None
-    else:
-        guess_next_dev = functools.partial(scm_version._guess_next_dev,
-                                           liberfadir=LIBERFADIR)
-
-use_scm_version = {
-    'write_to': os.path.join('erfa', '_version.py'),
-    'version_scheme': guess_next_dev,
```

Comment:
Not using `g.ess_next_dev` in `setup.py` would break the versioning scheme at the next update of the `liberfa` C library version,